spec/services/symphony_client_spec.rb

```diff
@@ -71,6 +93,84 @@
     end
   end
 
+  describe '#change_pickup_library' do
```

Comment:
Kinda curious as to why you added all of these SC specs for old methods. I didn't realize we were missing specs here. Thanks for adding them, I would definitely have done them though if you asked :). Looks like we are still missing one, `get_item_type_map`. I can take care of that one.